scripts/npmDepsHash.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+
+/* eslint-disable no-console */
+async function main(_argv = process.argv) {
+  const projectRoot = path.join(__dirname, '..');
+
+  childProcess.exec(
```

Comment:
This should be wrapped as a promise or made synchronous. Why is this not `execSync`?

Reply:
In fact the piping into `tail` is not the best way to do this.

You should have just processed the output in JS and used `execFileSync` instead as it is most secure and fastest performance.

Remember the reason to use a JS script was to be portable.

Reply:
Look https://github.com/MatrixAI/Polykey-CLI/blob/5a31aba123d855d585264caef7bba9119e535786/scripts/pkg.js#L101. This is better.

Reply:
Using execSync now.